### DISCUSSION TITLE
How to stop depth mask mesh inpainting when generating video midway through?

### DISCUSSION THREAD

**Discussion by vitalbuttering** [thygate/stable-diffusion-webui-depthmap-script#384] [General]
Accidentally clicked on a high res depth mask image to generate into a video but it is taking too long and decided to cancel it. unfortunately, have no idea if there's an option to do so in the UI. Is there? Or i just need to refresh the browser tab?

**Comment by sonakravtsenko3** [comment]
Just killing the process ("the console window") and restarting it always works. Simply reopening the tab will not make the process stop, although may allow doing other actions.

**Reply by vitalbuttering** [reply]
Yep that's what i did. Thanks!
